pkg/pipelinerun/description/description.go

```diff
@@ -207,6 +208,84 @@ func PrintPipelineRunDescription(s *cli.Stream, prName string, p cli.Params) err
 	return w.Flush()
 }
 
+// PrintPipelineRunStatus prints the status of the specific PilelineRun
+func PrintPipelineRunStatus(w io.Writer, prName string, p cli.Params) error {
+	cs, err := p.Clients()
+	if err != nil {
+		return fmt.Errorf("failed to create tekton client: %v", err)
+	}
+
+	pr, err := pipelinerun.Get(cs, prName, metav1.GetOptions{}, p.Namespace())
+	if err != nil {
+		return fmt.Errorf("failed to find pipelinerun %q", prName)
+	}
+
+	condition := formatted.Condition(pr.Status.Conditions)
+	_, err = fmt.Fprint(w, condition)
+	return err
+}
+
+// PrintPipelineRunStatusWithTaskRuns prints the status of the specific PilelineRun with associated taskruns
+func PrintPipelineRunStatusWithTaskRuns(s *cli.Stream, prName string, p cli.Params) error {
+	statusTempl := `{{decorate "status" ""}}{{decorate "underline bold" "Status"}}
+
+STATUS
+{{ formatCondition .PipelineRun.Status.Conditions }}
+
+{{decorate "taskruns" ""}}{{decorate "underline bold" "Taskruns\n"}}
```

Comment:
```suggestion
{{decorate "taskruns" ""}}{{decorate "underline bold" "Taskruns\n"}}
```
Not sure if this should be updated to TaskRuns to be consistent - would prob require additional changes.